source/server/server.cc

```diff
@@ -131,6 +131,7 @@ void InstanceImpl::failHealthcheck(bool fail) {
 
 void InstanceUtil::flushMetricsToSinks(const std::list<Stats::SinkPtr>& sinks,
                                        Stats::Source& source) {
+  ENVOY_LOG(debug, "flushing metrics");
```

Comment:
We already have debug log while flushing stats here
https://github.com/envoyproxy/envoy/blob/6674c03460b0dd0fe9a3bf3fce6eddfa98e1b1b0/source/server/server.cc#L143
Are you not getting it?

Reply:
make debug information more powerful

Reply:
If we want to add another debug log, I would probably add it here:
https://github.com/envoyproxy/envoy/blob/6674c03460b0dd0fe9a3bf3fce6eddfa98e1b1b0/source/server/server.cc#L147

Basically, when the histogram merge callback is complete.

Reply:
Yeah, Agree. While it is good to add more debug log lines, TBH I was not very clear the purpose of the log at the place added in this PR. Probably it would help to know, what prompted you to add the debug log so that as @mattklein123  suggested we can add the log at the right place or see if there is any other problem here - Did you see `flushing stats` line in the log and metrics not reaching the stats sinks so added this line to see if it is being called?